sockeye/data_io.py

```diff
@@ -1019,6 +1025,22 @@ def ids2strids(ids: Iterable[int]) -> str:
     return " ".join(map(str, ids))
 
 
+def ids2tokens(token_ids: Iterable[int],
```

Comment:
would that be better placed in utils.py where other token2id and vice versa methods are defined?

Reply:
Actually those are all defined in data_io; I put this right next to them.